parallel-state.js

```diff
@@ -0,0 +1,41 @@
+var Nanobus = require('nanobus')
+var assert = require('assert')
+
+module.exports = Parallelstate
+
+function Parallelstate (transitions) {
+  if (!(this instanceof Parallelstate)) return new Parallelstate(transitions)
+  assert.equal(typeof transitions, 'object', 'nanostate: transitions should be type object')
+
+  this.scopes = Object.keys(transitions)
+  this.transitions = transitions
+
+  this.scopes.forEach(scope => {
+    assert.ok(transitions[scope] instanceof Nanobus, `parallel expects "${scope}" to have nanostate as transition`)
```

Comment:
I have a concern about this assertion here. I'm not using `Nanostate` to check `instanceof`. Suggestions are welcome in a way to organize the code to use `Nanostate` instead of `Nanobus`. :)

Reply:
I think it'd be alright, if we don't assert here tbh. It would catch some errors, perhaps - but it's probably not really common that someone passes a wrong object in.

`instanceof` checks can get tricky fast, and because we're running this in a loop it also can't be removed statically. So I'm thinking perhaps the easiest resolution is to just remove it. Does that make sense?